src/modules/feedback/components/FeedbackModal.js

```diff
@@ -57,34 +57,44 @@ type PropsType = {|
   theme: ThemeType
 |}
 
+type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
+
 type StateType = {|
-  feedbackSent: boolean
+  feedbackSent: SendingStatusType
 |}
 
 export class FeedbackModal extends React.Component<PropsType, StateType> {
-  state = { feedbackSent: false }
+  state = { feedbackSent: 'IDLE' }
 
-  handleSubmit = () => this.setState({ feedbackSent: true })
+  handleSubmit = (value: SendingStatusType) => {
+    this.setState({ feedbackSent: value })
+  }
 
   handleOverlayClick = () => {
-    this.setState({ feedbackSent: false })
+    this.setState({ feedbackSent: 'IDLE' })
     this.props.closeFeedbackModal()
   }
 
   render () {
     const { theme, feedbackStatus, ...otherProps } = this.props
     const { feedbackSent } = this.state
 
+    let boxContainer
+    if (['IDLE', 'ERROR'].includes(feedbackSent)) {
```

Comment:
It feels kind of strange feedback errors are handled by the `FeedbackBoxContainer`. Imo it should
- either be handled by a separate component (similar to `FeedbackThanksMessage`)
- refactor `FeedbackThanksMessage` to a more generic component rendering different messages, i.e. a thanks message and an error message

Reply:
should be discussed generally and with regards to the NATIVE-App

Reply:
I thought the behavior in the native app was different, which is not the case. Since this implements the same behavior that is present in the native app, I'll resolve this. No further action required here.